app/src/main/java/org/xwiki/android/sync/activities/SettingServerIpViewFlipper.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.android.sync.activities
+
+import android.text.TextUtils
+import android.util.Log
+import android.view.View
+import android.widget.EditText
+import org.xwiki.android.sync.Constants
+import org.xwiki.android.sync.R
+import org.xwiki.android.sync.SERVER_ADDRESS
+import org.xwiki.android.sync.auth.AuthenticatorActivity
+import org.xwiki.android.sync.utils.SharedPrefsUtils
+import org.xwiki.android.sync.utils.putValue
+
+import java.net.MalformedURLException
+import java.net.URL
+
+/**
+ * Flipper for setting XWiki server address.
+ *
+ * @version $Id: 1764429bc21ab6d1aaa331ae01d81ac5b9afd1f2 $
+ */
+class SettingServerIpViewFlipper
+/**
+ * Standard constructor.
+ *
+ * @param activity Actual [AuthenticatorActivity]
+ * @param contentRootView Root [View] of that flipper (not activity)
+ */
+    (activity: AuthenticatorActivity, contentRootView: View) : BaseViewFlipper(activity, contentRootView) {
```

Comment:
I think, it is better to replace all constructors like this to the name of class, like `SomeClass(first: String) ...`. In Kotlin there is the thing like primary constructor. So, if you will replace documentation of the constructor into class definition docs, it will be ok.

Reply:
Understood.

Reply:
Can you, please, fix it for all constructors?